src/router/routes.ts

```diff
@@ -1,6 +1,6 @@
-import Home from '@/views/Home.vue'
-import MixinExample from '@/views/MixinExample.vue'
-import StateExample from '@/views/StateExample.vue'
+// import Home from '@/views/Home.vue'
+// import MixinExample from '@/views/MixinExample.vue'
+// import StateExample from '@/views/StateExample.vue'
 import DefineCompany from '@/views/DefineCompany.vue'
 import ReviewConfirm from '@/views/ReviewConfirm.vue'
```

Comment:
cool! Not sure how you feel about this, but in this UI it appears we will have a `View` for every step. So more than we did with COOPS-UI. Would it make sense for us to include an Index file for the views to clean up our Imports as the Views grow in number?

Reply:
Maybe but I'm not exactly sure where to go with this, because we'll have different views depending on the selected business type. So for now, my vote is don't do unknown work. What do you think?

Reply:
Totally, adding an index is 2 min job down the road, if we feel like we need one. Sounds good!